torch/quantization/fx/quantize.py

```diff
@@ -926,6 +954,87 @@ def get_next_i(module, qparams):
             delattr(observed_root, n)
         return GraphModule(observed_root, self.quantized_graph)
 
+    def _fold_weight(self, quantized):
+        def collect_nodes_to_fold(node):
+            nodes = [node]
+            frontier = [node]
+            while frontier:
+                node = frontier.pop()
+                all_args = list(node.args) + list(node.kwargs.values())
+                for arg in all_args:
+                    print('looking at arg:', arg)
```

Comment:
doesn't have to be for this PR, but should we have some kind of logging levels in the future?  This seems really useful when developing or debugging, and ideally we can keep it off by default so users don't see the low level.

Reply:
I think so, not sure what is the the recommended python logging for pytorch, I remember someone posted before and dima answered just use some global flag to control the logging level?